### PR TITLE
Fix to errors seen in stage after merge of last PR

### PR DESCRIPTION
**Pull request by ArielLima**
Prevent erroring out from attempted account cleanup. Instead logging the error and continuing to account initialization normally.

### CODE REVIEW

**Review by fahlmant — APPROVED** [review]
/lgtm

**Review by fahlmant — APPROVED** [review]
/lgtm

File: pkg/controller/account/ec2.go [hunk 1]
```diff
@@ -88,8 +88,10 @@ func (r *ReconcileAccount) InitializeRegion(
 	reqLogger.Info("initializing region", "region", region)
 
 	// Attempt to clean the region from any hanging resources
-	cleaned, err := cleanRegion(awsClient, reqLogger)
+	cleaned, err := cleanRegion(awsClient, reqLogger, account.Name)
 	if err != nil {
+		cleanErr := fmt.Sprintf("Error", err.Error(), "account", account.Name)
```
Comment (rogbas) [inline pkg/controller/account/ec2.go:93]: @ArielLima I suggest having a more descriptive msg than Error here.

The previous written msg would work fine IMO, and then you could append the aws err string as you're doing

**Pull request merged**